top500_plot.py

```diff
@@ -18,12 +18,15 @@
 
 # get the data
 df = pd.read_csv('TOP500_history.csv', low_memory=False, parse_dates={'Date': ['Year','Month','Day']})
-assert (df.groupby(('Date',)).size()==500).all()
+assert (df.groupby(('Date')).size()==500).all()
 
 # Make mostly-coherent processor family and vendor columns
 def remap(procfam):
-    if procfam in ('Intel EM64T','Intel Nehalem','Intel Westmere','Intel SandyBridge','Intel IvyBridge','Intel Haswell','Intel Core','Intel Broadwell','Intel Skylake','Intel Cascade Lake','Intel Cascade lake','AMD x86_64','AMD Zen (Naples)','AMD Zen-2 (Rome)'):
-        i,v='x86-64', procfam.split()[0]
+    if procfam in ('Intel EM64T','Intel Nehalem','Intel Westmere','Intel SandyBridge','Intel IvyBridge','Intel Haswell','Intel Core','Intel Broadwell','Intel Skylake','Intel Cascade Lake','Intel Cascade lake','Intel Ice Lake','X86_64','AMD x86_64','AMD Zen (Naples)','AMD Zen-2 (Rome)','AMD Zen-3 (Milan)'):
+        if procfam.split()[0] == 'X86_64':
+            i,v='x86-64', "AMD" # AMD's "Hygon Dhyana" are mis-classified
```

Comment:
So these show up with a "raw" `procfam` value of `X86_64`… nothing more and nothing less? (Yay, TOP500 data quality… 😒)

If so, can you can move this to a separate clause for clarity? It doesn't really fit in with this clause, which is intended for `procfam` values where the first word is `Intel` or `AMD`. Something like…

```py
elif procfam == 'X86_64':
    i,v='x86-64', 'Hygon Dhyana'
```